src/main/java/calculator/Main.java

```diff
@@ -0,0 +1,56 @@
+package calculator;
+
+import calculator.domain.calculator.Calculator;
+import calculator.domain.calculator.NonNumericValuesException;
+import calculator.domain.calculator.StringCalculator;
+import calculator.domain.splitter.CommaAndColonSplitter;
+import calculator.domain.splitter.CustomSplitter;
+import calculator.domain.splitter.Splitter;
+
+import java.util.Arrays;
+
+public class Main {
+    private static final int ZERO = 0;
+
+    public static int calculate(String input) {
+
+        if (emptyCheck(input)) {
+            return ZERO;
+        }
+
+        String[] splitInput = split(input);
+        validateNonNumericValues(splitInput);
+
+        Calculator stringCalculator = new StringCalculator(splitInput);
+
+        return stringCalculator.sum();
+    }
+
+    private static boolean emptyCheck(String input) {
+        return input == null || input.isEmpty();
```

Comment:
```suggestion
        return Objects.isNull(input) || input.isEmpty();
```
 Objects.isNull()을 사용해보는것이 어떨까요 ?